Kernel/Prekernel/init.cpp

```diff
@@ -73,6 +73,10 @@ extern "C" [[noreturn]] void init()
         halt();
 
     multiboot_module_entry_t* kernel_module = (multiboot_module_entry_t*)(FlatPtr)multiboot_info_ptr->mods_addr;
+    for (size_t i = 0; i < multiboot_info_ptr->mods_count; i++) {
+        if (kernel_module[i].start & (PAGE_SIZE - 1))
+            halt();
+    }
```

Comment:
Is this part of the DROW simplification? It was not really mentioned in the commit message and there's no comment explaining its purpose.

Reply:
Ah, I see, this is how the DROW is loaded. Would be nice to have a comment about why this check should hold. :)